Source/cmake/OptionsCommon.cmake

```diff
@@ -154,7 +154,7 @@ if (USE_THIN_ARCHIVES)
 endif ()
 
 set(ENABLE_DEBUG_FISSION_DEFAULT OFF)
-if (CMAKE_BUILD_TYPE STREQUAL "Debug")
```

Comment:
Remove this line and its corresponding `endif()`, then dedent the contents of the block. That will always pass `-gsplit-dwarf` to the compiler flags regardless of the build type, which is a better option [as outlined here](https://github.com/WebKit/WebKit/pull/9602#discussion_r1098443968).